verl/utils/dataset/rl_dataset.py

```diff
@@ -300,11 +300,13 @@ def __getitem__(self, item):
         # add index for each prompt
         index = row_dict.get("extra_info", {}).get("index", 0)
         tools_kwargs = row_dict.get("extra_info", {}).get("tools_kwargs", {})
+        interaction_kwargs = row_dict.get("extra_info", {}).get("interaction_kwargs", {})
```

Comment:
Could you add a config to control whether to include the tool related fields in dataproto? This may increase the size if tools are not used

Reply:
https://github.com/volcengine/verl/issues/2145 Tracked by new issue, will refactor in future.